Add GCP disk and Cloud SQL playbooks (#1720)
sourcegraph/about: handbook/engineering/incidents/playbooks/index.md

If you discover an issue with Netlify but their status page is not displaying it, open a support request using the credentials in 1Password.

Sites:

- `about.sourcegraph.com`
- `sourcegraph.com/about` -307-> `about.sourcegraph.com/about`
- `sourcegraph.com/pricing` -307-> `about.sourcegraph.com/pricing`

For a full list of sites redirected from `sourcegraph.com/$KEY` reference the [router code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/cmd/frontend/internal/app/ui/router.go#L86) in our frontend.

## Moving a GCE disk between projects

Moving a disk between projects requires multiple steps to ensure the disk is detached from its original source.

- The initial disk can only be created using the same type. To change its type, you will have to snapshot the disk and restore as a new type.

> - The initial disks can only be moved to the same region as in their source project.

Reviewer: ```suggestion
- The initial disks can only be moved to the same region as their source project.
```

```

We now have to snapshot and restore the disk, to ensure the disk is unlinked from its source disk. This process is the same as its used to [change a disk zone or type](#changing-a-gce-disk-zone-or-type), please follow that playbook.

## Changing a GCE disk zone or type

To change disk region or type, we need to first create a snapshot and then restore it to the new region or type.

Snapshot:

```
gcloud compute disks snapshot ${SOURCE_DISK_NAME} --project=${PROJECT} --zone=${SOURCE_ZONE} --snapshot-names ${SNAPSHOT_NAME}
```

At this point, you can optionally delete the source disk.

Restore:

```
gcloud compute disks create ${TARGET_DISK_NAME} --project=${PROJECT} --size ${TARGET_DISK_SIZE} --source-snapshot ${SNAPSHOT_NAME} --zone=${TARGET_ZONE} --type ${TARGET_DISK_TYPE}
```

_TARGET_DISK_SIZE has to be equal or larger than the original source disk._

There is an [alias command](https://cloud.google.com/sdk/gcloud/reference/compute/disks/move) that performs the `snapshot -> restore` operation automatically to switch regions, but it fails on big disks.

## Linking a PV/PVC to a GCE disk

> When restoring a disks from a snapshot, or if for other reasons we need to re-link a disk to a PersistentVolume and its associated PersistentVolumeClaim, we will need to adapt the Kubernetes resource definitions to link the existing GCE disk to GKE.

Reviewer: ```suggestion
When we need to re-link a disk to a PersistentVolume and its associated PersistentVolumeClaimWhen because we are restoring a disks from a snapshot or for some other reasons, we will need to adapt the Kubernetes resource definitions to link the existing GCE disk to GKE.
```

Author: There seems to be a typo here in `PersistentVolumeClaimWhen`, what was it meant to say?

Reviewer: copy/paste fat finger :-) 

```suggestion
When we need to re-link a disk to a PersistentVolume and its associated PersistentVolumeClaim because we are restoring a disks from a snapshot or for some other reasons, we will need to adapt the Kubernetes resource definitions to link the existing GCE disk to GKE.
```

Reviewer: just a suggestion from a non-native English speaker. feel free to ignore :-)
